### DISCUSSION TITLE
how does UPLOAD_NAME_USERNAME work?

### DISCUSSION THREAD

**Discussion by inanme** [goreleaser/goreleaser#1939] [Q&A] [answered]
Hi, I can't seem to make this work. This is from https://goreleaser.com/customization/upload/

  The name of the environment variable will be UPLOAD_NAME_USERNAME.
  The name of the environment variable will be UPLOAD_NAME_SECRET.

What I am trying is:
`UPLOAD_NAME_USERNAME=MY_USERNAME UPLOAD_NAME_SECRET=MY_PASSWORD goreleaser release`

Many thanks

**Comment by caarlos0** [comment, marked as answer]
`NAME` should be the same as the name you put in the config section...

for example, if:

```yaml
- name: foo
```

the envs will be `UPLOAD_FOO_USERNAME` and `UPLOAD_FOO_SECRET`.
